docs/snapshotters/erofs.md

```diff
@@ -0,0 +1,143 @@
+# EROFS Snapshotter
+
+The [EROFS](https://erofs.docs.kernel.org) snapshotter is an experimental
+feature, which is able to leverage EROFS-formatted blobs for each committed
+snapshot and prepares an EROFS + OverlayFS mount for each active snapshot.
+
+In order to leverage EROFS-formatted blobs, the EROFS differ is needed to be
+used together to apply image layers.  Otherwise, the EROFS snapshotter will
```

Comment:
AFAUI, this handles the whiteouts also. perhaps add a note about it (and how it's done (in `mkfs.erofs`?).

Reply:
> AFAUI, this handles the whiteouts also. perhaps add a note about it (and how it's done (in `mkfs.erofs`?).

 I could try to mention it a bit, but that is an implementation detail, since it's a real necessary part to support OCI images. `--aufs` option in `mkfs.erofs` will handle AUFS-like whiteouts and opaques properly.